src/lib/factories.js

```diff
@@ -105,8 +114,8 @@ export function createShorthandFactory(Component, mapValueToProps, generateKey)
     throw new Error('createShorthandFactory() Component must be a string or function.')
   }
 
-  return (val, defaultProps) => {
-    return createShorthand(Component, mapValueToProps, val, defaultProps, generateKey)
+  return (val, defaultProps, overrideProps) => {
+    return createShorthand(Component, mapValueToProps, val, defaultProps, overrideProps, generateKey)
```

Comment:
The only comment I have is that the arguments here are getting a little wild.  There are cases where you'd want to set `generateKey` but not `defaultProps` nor `overrideProps`.  That usage is ugly:

```jsx
Menu.create = createShorthandFactory(
  Menu, 
  items => ({ items }),
  null, // defaultProps
  null, // overrideProps
  true, // generateKey
)
```

Perhaps we move these optional arguments to an options object?

```jsx
Menu.create = createShorthandFactory(
  Menu, 
  items => ({ items }), 
  { generateKey: true }
)
```